ldms/src/ldmsd/ldmsd_updtr.c

```diff
@@ -537,28 +548,35 @@ void __ldmsd_prdset_lookup_cb(ldms_t xprt, enum ldms_lookup_status status,
 		assert(NULL == set);
 		status = (status < 0 ? -status : status);
 		if (status == ENOMEM) {
-			ldmsd_log(LDMSD_LERROR,
-				"Error %d in lookup callback for set '%s' "
-				"Consider changing the -m parameter on the "
-				"command line to a larger value. "
-				"The current value is %s\n",
-				status, prd_set->inst_name,
-				ldmsd_get_max_mem_sz_str());
+			if (! (prd_set->warned & LDMSD_PRDCR_SET_WARNED_NOMEM)) {
```

Comment:
You're daemon is basically hosed at this point. Hiding these messages will make it seem as if everything is ok, when in fact, these sets are being ignored.

Reply:
We are not hiding the messages, we're simply emitting each message once per bad set.
The goal here is to prevent redundantly flooded logs, not hiding the messages.